src/Com/NoSyn/Parser/ConcreteSyntaxConverter.hs

```diff
@@ -109,7 +108,8 @@ convertParameter :: CParameter -> CompilerStatus Ifm1Parameter.Parameter
 convertParameter (CParam a b) = return $ Ifm1Parameter.IfParameter $ PConst a b
 convertParameter (CPointerParam a op b)
     | op == "*" = return $ Ifm1Parameter.IfParameter $ PPointer a b
-    | otherwise = Error "Only '*' can be used on a type to denote an operator" (show (CPointerParam a op b))
+    | op == "..." = return $ Ifm1Parameter.IfParameter $ PVariadic a b
```

Comment:
#21 Similar approach can be taken here as to what allow as a type modifying operator